conda/plugins/hookspec.py

```diff
@@ -301,3 +301,35 @@ def conda_post_solves():
                    action=example_post_solve,
                )
         """
+
+    @_hookspec
+    def conda_configuration_parameters(self) -> Iterable[CondaPostSolve]:
```

Comment:
Also needs an import

```suggestion
    def conda_configuration_parameters(self) -> Iterable[CondaConfigurationParameter]:
```